notes/force-move-spec.md

```diff
@@ -0,0 +1,208 @@
+# Force Move Spec
+
+To serve as a succinct specification of the ForceMove protocol.
+Everthing required for getting to an outcome.
+
+Things have changed since the original paper.
+Names have changed. Optimizations have been added.
+This should serve as the definitive version over time.
+
+Not a concrete specification.
+Abstract notion of "the state of the channel".
+Don't specify formats.
+In methods, we provide the minimal required parameters.
+In a concrete implementation, the parameters might change - for example it can be more efficient
+
+## Off-chain state
+
+| **Field**      | **Data type**                 | **Definition / Explanation**                                                                         |
+| :------------- | :---------------------------- | :--------------------------------------------------------------------------------------------------- |
+| ChainID        | `bytes32`                     | e.g. ropsten / mainnet                                                                               |
+| Participants   | `address[]`                   | addresses of participants. Determines keys used to sign updates                                      |
+| ChannelNonce   | `uint256`                     | unique, to prevent replay attacks                                                                    |
+| TurnNum        | `uint256`                     | turn number                                                                                          |
+| DefaultOutcome | `[address, uint256][]`        | tracks the amounts paid out to each of a list of addresses if the channel is finalized in this state |
+| isFinal        | `boolean`                     |                                                                                                      |
```

Comment:
Can we use the word Terminal (i.e., `isTerminal`)

Reply:
Maybe. Let's discuss this too. This needs to fit with the concept of finalizability.

Reply:
What does this flag mean? What does "final" mean in the rest of the specs?

Reply:
If you have a sequence of `n` states, which all have `isFinal` set, then you can can call the `conclude` method and finalize the outcome instantly.

Reply:
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll yeah it's equivalent to `isStateTerminal()` returning `true` in `CounterfactualApp`

Reply:
@snario I don't think they're equivalent...

Reply:
@tomclose just type-checking the terminology, a state can be final and an outcome can be final, but these mean different things, correct?

Reply:
`isFinal`